chart/values.yaml

```diff
@@ -107,6 +107,8 @@ airflowPodAnnotations: {}
 
 # Enable RBAC (default on most clusters these days)
 rbacEnabled: true
+# What kind of rbac should be used (Role or ClusterRole)
```

Comment:
I think you can easilly raise if not on authorized values, though it might do it on itself already
https://stackoverflow.com/questions/61496709/helm-template-how-raise-exception-in-in-helm-function

Reply:
They have some dynamic checks on the values.yaml with a json schema in the newer version of the chart. I'll do it after rebase :)